src/Saga.php

```diff
@@ -11,13 +11,16 @@
 
 namespace Broadway\Saga;
 
+use Broadway\Domain\DomainMessage;
+
 abstract class Saga implements SagaInterface
 {
     /**
      * {@inheritDoc}
      */
-    public function handle($event, State $state)
+    public function handle(State $state, DomainMessage $domainMessage)
```

Comment:
I prefer to pass the DomainMessage first, the State second. Your opinion, @broadway/broadway-saga ?